src/Facades/LetsEncrypt.php

```diff
@@ -8,10 +8,10 @@
 
 /**
  * @mixin \Daanra\LaravelLetsEncrypt\LetsEncrypt
- * @method static array create(string $domain)
+ * @method static array create(string $domain, array $chain = [])
  * @method static LetsEncryptCertificate createNow(string $domain)
- * @method static LetsEncryptCertificate renewNow($domain)
- * @method static PendingDispatch renew($domain)
+ * @method static LetsEncryptCertificate renewNow(string $domain)
```

Comment:
`renew` and `renewNow` may also accept a `LetsEncryptCertificate` for `$domain` so its signature should be `string|LetsEncryptCertificate`

Reply:
Yes, exactly!